frontend/src/app/main/data/dashboard.cljs

```diff
@@ -171,8 +179,10 @@
     (ptk/reify ::fetch-shared-files
       ptk/WatchEvent
       (watch [_ state stream]
-        (->> (rp/query :shared-files {:team-id team-id})
-             (rx/map #(partial fetched %)))))))
+        (rx/concat
+          (->> (rp/query :shared-files {:team-id team-id})
+               (rx/map #(partial fetched %)))
+          (rx/of (clear-selected-files)))))))
```

Comment:
I think that clear-selected-files is a side effect that should be done by the caller and not on this event. I'mean, this event is for fetching data, not clearing selection.

Reply:
Done. You can squash the commit into the first one when merging.